jscomp/build_tests/super_errors/expected/dict_coercion.res.expected

```diff
@@ -0,0 +1,10 @@
+
+  [1;31mWe've found a bug for you![0m
+  [36m/.../fixtures/dict_coercion.res[0m:[2m7:10-30[0m
+
+  5 [2m│[0m type fakeDict<'t> = {dictValuesType?: 't}
+  6 [2m│[0m 
+  [1;31m7[0m [2m│[0m let d = ([1;31mdict :> fakeDict<int>[0m)
+  8 [2m│[0m 
+
+  Type Js.Dict.t<int> = dict<int> is not a subtype of fakeDict<int>
```

Comment:
why not? Should the message say a bit more?

Reply:
Definitely. But that's a separate, and large, task.